predict.py

```diff
@@ -274,41 +324,35 @@ def compile_fp8(self):
 
         print("compiled in ", time.time() - st)
 
-    def compile_bf16(self):
-        print("compiling bf16 model")
-        st = time.time()
-
-        self.compile_run = True
-        self.base_predict(
-            prompt="a cool dog",
-            aspect_ratio="1:1",
-            num_outputs=1,
-            num_inference_steps=self.num_steps,
-            guidance=3.5,
-            seed=123,
-        )
-        print("compiled in ", time.time() - st)
-
-    def aspect_ratio_to_width_height(self, aspect_ratio: str):
-        return ASPECT_RATIOS.get(aspect_ratio)
-
-    def get_image(self, image: str):
-        if image is None:
-            return None
-        image = Image.open(image).convert("RGB")
-        transform = transforms.Compose(
-            [
-                transforms.ToTensor(),
-                transforms.Lambda(lambda x: 2.0 * x - 1.0),
-            ]
-        )
-        img: torch.Tensor = transform(image)
-        return img[None, ...]
-
-    def get_mask(self, image: str):
-        if image is None:
-            return None
-        image = Image.open(image).convert("L")
+    # TODO(andreas): This is never used and will throw an error if
+    # used because aspect_ratio is not a valid argument to bf16_predict
```

Comment:
we could fix it with `width=1024, height=1024`, but it remains unused

Reply:
I say we just delete this; commented out code rots and this is already unused, we can pull this out of git history if we want to resurrect.